tokio/src/runtime/task/trace/mod.rs

```diff
@@ -82,14 +82,18 @@ impl Context {
     where
         F: FnOnce(&Self) -> R,
     {
-        crate::runtime::context::with_trace(f)
+        unsafe { crate::runtime::context::with_trace(f) }
```

Comment:
This function is very short, and it’s clear enough for developers to reason about the safety requirements, so we don’t need additional "Safety" comments here”